.prout.json

```diff
@@ -2,7 +2,10 @@
     "checkpoints": {
         "PROD": {
             "url": "https://frontend.gutools.co.uk/deploys?stage=PROD&pageSize=50",
-            "overdue": "30M"
+            "overdue": "30M",
+            "messages": {
+                "seen": "prout/seen.md",
```

Comment:
@TBonnin there's a trailing comma here, which Prout's JSON parser does not like!

```
play.api.UnexpectedException: Unexpected exception[JsonParseException: Unexpected character ('}' (code 125)): was expecting double-quote to start field name 
Jun 13 11:58:26 prout-bot app/web.1:  at [Source: [B@7b8afdb3; line: 8, column: 14]] 
Jun 13 11:58:26 prout-bot app/web.1: 	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:261) 
```